diagnostic.md

```diff
@@ -6,70 +6,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+before_action :set_patient, only: [:show]
+def show
+    render json: @patient
+end
+
+def set_patient
+    @patient = Patient.find(params[:id])
+end
+private :set_patient
+
+def show
+  @patient = Patient.find(params[:id])
+
+  render json: @patient
+end
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+In the `app/controllers/books_controller.rb`.
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+The command `bundle exec rake db:migrate` creates a path for the developer to
```

Comment:
`It executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.`